aten/src/ATen/cpu/vec/sve/vec_double.h

```diff
@@ -484,11 +484,12 @@ Vectorized<double> inline Vectorized<double>::le(const Vectorized<double>& other
 template <>
 inline void convert(const double* src, double* dst, int64_t n) {
   const int64_t fraction = n % Vectorized<double>::size();
+#ifndef __msvc_cl__
 #pragma unroll
+#endif
```

Comment:
would it be better to use c10::ForcedUnroll?

Reply:
Thanks for this suggestion. Updated the PR.